ci/build.sh

```diff
@@ -150,13 +150,13 @@ build_docker_image "${DOCKER_TAG}" || (echo "Error during building docker image"
 
 if [ -n "${DOCKER_PASSWORD}" ] && [ -n "${TRAVIS_TAG}" ]; then
   push_docker_image "${VERSION}"
-  push_helm_chart "${VERSION}"
+  push_helm_chart "${VERSION}" "./"
```

Comment:
```suggestion
  push_helm_chart "${VERSION}" "."
```

Reply:
It seems that it doesn't work without / but it could be "" instead of "./". What do you think?

Reply:
Could you add "/" to the line 113? I edited the suggestion but too late as I see 😅

Reply:
Ok, I will add "/" in line 113.